samples/bmp280-sensor-azure-iot-hub/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
```

Comment:
You are missing the standard // header. Please look at the other files, you'll find it in any cs file!

Reply:
Sure, added, thanks